src/components/Tag/Tag.tsx

```diff
@@ -30,8 +30,9 @@ const Tag = ({
     >
       <span
         sx={{
-          variant: "text.caption",
+          cursor: "default",
           fontWeight: "bold",
+          variant: "text.caption",
```

Comment:
`fontWeight` has to be after so it's not overwritten by the variant font weight.
It's breaking the Tag storyshots as it is now.
```suggestion
          variant: "text.caption",
          fontWeight: "bold",
```